extension/src/processExecution.ts

```diff
@@ -58,3 +59,6 @@ export const executeProcess = async (
   const { stdout } = await createProcess(options)
   return stdout
 }
+
+export const processExists = (pid: number): Promise<boolean> =>
```

Comment:
[F] Gives type safety that we pass a number to the original `processExists`. This is important because (at least on M1) giving the string version of a valid and running numeric pid returns `false` 😱